src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+const totalPopulationEl = document.querySelector('.total-population');
+const averagePopulationEl = document.querySelector('.average-population');
+
+const totalPopulation = [...populations].reduce((sum, population) => {
+  return sum + (Number(population.textContent.split(',').join('')) || 0);
+}, 0);
+
+totalPopulationEl.innerHTML = `${numberWithCommas(totalPopulation)}`;
+
+averagePopulationEl.innerHTML = `${numberWithCommas(
+  Math.round(totalPopulation / populations.length),
+)}`;
```

Comment:
There's a trailing comma after `Math.round(totalPopulation / populations.length)`. In JavaScript, trailing commas in function arguments are generally not recommended and can cause issues in some environments. Consider removing it.